tests/testthat/test-wildcards.R

```diff
@@ -24,5 +24,35 @@ test_that("HTML `data-*` & `aria-* ` wildcards are passed along to layout approp
   expect_equal(x$props$`data-foo`, 1)
 })
 
+
+test_that("restricted pattern matching callback selectors must be formatted as a list", {
+  app <- Dash$new()
+  app$layout(htmlDiv(list(
+    htmlButton("Add Filter", id="add-filter", n_clicks=0),
+    htmlDiv(id="dropdown-container", children=list()),
+    htmlDiv(id="dropdown-container-output")
+  )))
+  
+  expect_error(
+    app$callback(
+      output(id="dropdown-container-output", property="children"),
+      params = list(
+        input(id=list("index" = dash:::ALL, "type" = "filter-dropdown"), property= "value")
+      ),
+      display_output <- function(test){
+        return(htmlDiv(
+          lapply(1:length(test), function(x){
+            return(htmlDiv(test))
+          })
+        ))
+      }
+    ),
+    "Error: A callback input ID contains restricted pattern matching callback selectors ALL, 
+    MATCH or ALLSMALLER. Please verify that it is formatted as a pattern matching callback 
+    list ID, or choose a different component ID.",
```

Comment:
Related to earlier comment, wonder if we still need this test now that we're using symbols rather than character strings.

Reply:
Removed in 577deb4.